docs/references.rst

```diff
@@ -47,6 +47,8 @@ References
            Division, NWS Southern Region Headquarters, 1990.
            `SR90-23 <http://www.weather.gov/media/ffc/ta_htindx.PDF>`_
 
+.. [Salby1996] Salby, Murray L., 1996: Fundamentals of Atmospheric Physics.        
```

Comment:
Trailing whitespace is what's killing the build. Fix that up and we can help you clean the commit history and this should be good to go.